docs/detailed-push-process.md

```diff
@@ -63,16 +59,13 @@ myapplication.my_epinio_system_domain.com
 
 You can get the route of your application with `epinio apps list` or `epinio apps show myapplication`
 
-## 6. Additional things
+## Additional things
 
-During installation, if you specified a system domain using the `--system-domain` parameter, then your application routes will be sudomains of that domain.
-Epinio considers this domain to be a production server and thus creates a production level TLS certificate for your application using [Let's Encrypt](https://letsencrypt.org/).
-This happens by using the [cert-manager](https://cert-manager.io/docs/installation/kubernetes/) which is one more of the components Epinio installs with `epinio install`.
+During installation, if you specified a system domain using the `--system-domain` parameter, then your application routes will be subdomains of that domain.
+Epinio considers this domain to be a production server and thus creates a production level TLS certificate for your application using [Let's Encrypt](https://letsencrypt.org/). This happens by using the [cert-manager](https://cert-manager.io/docs/installation/kubernetes/) which is one more of the components Epinio installs with `epinio install`.
```

Comment:
`This happens by using the ... which` :arrow_right: `This is done by the ..., which`.
Note the `,` I added before `which`.